src/tokenizer/index.js

```diff
@@ -613,7 +613,7 @@ export default class Tokenizer {
       if (ch === quote) break;
       if (ch === 92) { // '\'
         out += this.input.slice(chunkStart, this.state.pos);
-        out += this.readEscapedChar(false);
+        out += this.readEscapedChar(false, true);
```

Comment:
Note: this is a good example of what I explained above. Reading line says nothing on what the function actually does.